skrifa/src/outline/cff/hint.rs

```diff
@@ -294,8 +297,8 @@ impl HintState {
             }
             if !zone.is_bottom
                 && top_edge.is_top()
-                && (zone.cs_bottom_edge - fuzz) <= top_edge.cs_coord
-                && top_edge.cs_coord <= (zone.cs_top_edge + fuzz)
+                && zone.cs_bottom_edge.wrapping_sub(fuzz) <= top_edge.cs_coord
+                && top_edge.cs_coord <= zone.cs_top_edge.wrapping_add(fuzz)
             {
                 // Top edge captured by top zone.
                 adjustment = if self.suppress_overshoot {
```

Comment:
Changed 285 and 306. I'm less concerned about the device space derived values.

We can be more defensive, but there are also numerous other places throughout this file where we have potential overflow panics from addition and subtraction. My personal preference would be to change the `Fixed` ops directly rather than making them all explicitly wrapping which just seems like noise.

Reply:
Ack, sgtm.